### PR TITLE
Embedded parent wallet section addition

### PR DESCRIPTION
**Pull request by moskalyk**

### CODE REVIEW

File: docs/pages/solutions/builder/embedded-wallet/parent-wallet.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Embedded Wallet Parent Wallet
```
Comment (taylanpince) [inline docs/pages/solutions/builder/embedded-wallet/parent-wallet.mdx]: I think we can remove this page @moskalyk What do you think? You've covered the same info on the main overview page.

File: docs/pages/solutions/builder/embedded-wallet/index.mdx [hunk 2]
```diff
@@ -26,12 +33,29 @@ We recommend that:
 
 Once the recovery wallet is setup properly, provide the public address for it. Recovery wallet cannot be modified once your configuration is saved.
 
-## Initial Configuration and Update Code
-Once you have all information prepared, go ahead and create your Embedded Wallet configuration. During initial setup, Builder will provide you with an update code. This code will be required for all subsequent modifications to your Embedded Wallet configuration, so it must be stored safely.
-
 ## Configuration Changes
 To make modifications to your Embedded Wallet configuration, navigate to the Embedded Wallet section again and make the necessary changes. Then click "Save Configuration" and enter your update code to deploy the changes.
```
Comment (taylanpince) [inline docs/pages/solutions/builder/embedded-wallet/index.mdx]: Can you also change this to reflect that we now call this "password" and not "update code". And also how important it is to store the password securely? If the password is lost we can't help them update their configuration.

**Review by taylanpince — CHANGES_REQUESTED** [review]

File: docs/pages/solutions/builder/embedded-wallet/parent-wallet.mdx (hunk 1 above)
Comment (moskalyk) [reply docs/pages/solutions/builder/embedded-wallet/parent-wallet.mdx]: sounds good

File: docs/pages/solutions/builder/embedded-wallet/index.mdx (hunk 2 above)
Comment (moskalyk) [reply docs/pages/solutions/builder/embedded-wallet/index.mdx]: Okay added back in

**Review by taylanpince — APPROVED** [review]

**Pull request merged**